extensions/wikia/ImageLazyLoad/js/ImgLzy.module.js

```diff
@@ -67,8 +66,15 @@ define('wikia.ImgLzy', ['jquery', 'wikia.log', 'wikia.window'], function ($, log
 
 		// rewrite the URL to request WebP thumbnails (if enabled on this wiki and supported by the browser)
 		rewriteURLForWebP: function (src) {
-			if (w.wgEnableWebPThumbnails === true && this.browserSupportsWebP && thumbCheckRegExp.test(src)) {
-				src = src.replace(/\.[^\./]+$/, '.webp');
+			if (w.wgEnableWebPThumbnails === true && this.browserSupportsWebP && thumbExtCheckRegExp.test(src) && thumbnailer.isThumbUrl(src)) {
+				if (thumbnailer.isLegacyThumbnailerUrl(src)) {
+					src = src.replace(/\.[^\./]+$/, '.webp');
+				}
+				else {
+					// add "format=webp" to Vignette URL
+					src += src.indexOf('?') ? '&' : '?';
+					src += 'format=webp';
```

Comment:
Oh, yeah - good question.

What are the cases when `format` parameter is set? Should we replace its value with `format` or keep the existing one and simply skip "webp-ization" here?